package.json

```diff
@@ -1,22 +1,21 @@
 {
-    "name": "{{ project-name }}",
-    "version": "1.0.0",
-    "description": "A template for kick starting a Cloudflare Workers project",
-    "main": "index.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "format": "prettier --write '**/*.{js,css,json,md}'"
-    },
-    "author": "{{ authors }}",
-    "license": "MIT",
-    "devDependencies": {
-        "husky": "^2.1.0",
-        "prettier": "^1.17.0"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "npm run format",
-            "pre-push": "npm run format"
-        }
+  "version": "1.0.0",
+  "description": "A template for kick starting a Cloudflare Workers project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write '**/*.{js,css,json,md}'"
```

Comment:
I don’t think that this will use your eslint rules at all, just prettier. Could be wrong though